pages/vi/faq.md

```diff
@@ -66,6 +67,20 @@
     - Name the organization '&lt;YourUserName&gt;-OLE', select 'My personal account', choose the 'Team For Open Source' option, and enter your email address to create the account.
     - Now it will prompt you for which profile to use when you fork the [treehouses repository](https://github.com/treehouses/treehouses.github.io).
 
+#### Q11: What if you encounter Host key verification fail?
+
++ To fix the issue, please type  `rm YourHomeDirectory/.ssh/known_hosts` in your powershell or terminal.
```

Comment:
in the 10 steps we support only `bash` driven shells
- macOs: console (...)
- windows: git bash (cygwin, wsl and wsl2 with debian)
- ubuntu: terminal (konsole, ... )

Reply:
so please reduce the message to only terminal syntax ;)